Kenneth-Macharia/Store-Manager-api
# #161267755 Get all endpoint

What does this PR do?
-Adds GET/ product endpoint.

Description of Task to be completed?
-Adds functionality for the GET/ product endpoint.
-Ensure the GET/ product TDD passes.

How should this be manually tested?
Clone this repo and cd into it.
-Activate the virtual environment 'env' at the prompt.
-Install dependencies: 'pip install -r requirements.txt
-Run pytest at the prompt: 'py.test'
-Test results should be as shown below:

![screen shot 2018-10-17 at 12 08 45 am](https://user-images.githubusercontent.com/42747960/47047684-e57dce80-d1a0-11e8-9d2c-1f3839c8c1bb.png)
